src/box/iproto.cc

```diff
@@ -2989,6 +3035,8 @@ iproto_stats_get_for_thread(struct iproto_stats *stats, int thread_id)
 	assert(thread_id >= 0 && thread_id < iproto_threads_count);
 	cfg_msg.stats = stats;
 	iproto_do_cfg(&iproto_threads[thread_id], &cfg_msg);
+	stats->requests_in_progress =
+		iproto_threads[thread_id].tx.requests_in_progress;
 }
```

Comment:
It looks like you could rewrite iproto_stats_get() using iproto_stats_get_for_thread():

```c
void
iproto_stats_get(...)
{
    for (int i = 0; i < iproto_threads_count; i++) {
        iproto_stats_get_for_thread(...);
        iproto_stats_add(...);
    }
}
```

Reply:
fixed